src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -12,10 +12,13 @@
   // rate of failing to fire torpedos [0.0, 1.0]
   private double FAILURE_RATE = 0.0; //NOSONAR
 
+  Random generator;
```

Comment:
I would prefer a static Random, but this is also good.

Reply:
Can you do it please?

Reply:
OK, I will.